### ISSUE TITLE
Need a trackbar control mode

### ISSUE THREAD

**Issue by wongshek** [shg8/3DGS.cpp#22]
Could you add a trackbar control mode to this great viewer, just like meshlab~
The default mouse and keyboard control modes are a bit uncomfortable to use now :(

**Comment by shg8** [comment]
Do you mean a trackball? There [this](https://github.com/BrutPitt/imGuIZMO.quat) imGUI library that I should be able to integrate fairly easily if it helps your use case.

**Comment by wongshek** [comment]
Yes, it's trackball, not trackbar. imGUI library is exactly what I need, it would be great if it could be integrated, thank you very much :)
